binaries/dump_operator_names.cc

```diff
@@ -0,0 +1,54 @@
+#include <torch/csrc/jit/script/module.h>
+#include <torch/csrc/jit/mobile/import.h>
+#include <torch/csrc/jit/mobile/module.h>
+#include <torch/csrc/jit/import.h>
+#include <torch/csrc/jit/export.h>
+#include <torch/csrc/jit/instruction.h>
+#include <torch/script.h>
+
+#include <fstream>
+
+namespace torch {
+namespace jit {
+void dump_opnames(const script::Module& m, std::unordered_set<std::string>& opnames) {
+  auto methods = m.get_methods();
+  for (const auto &method : methods) {
+    const auto &func = method.function();
```

Comment:
nit: "const auto &" -> "const auto&", to be consistent with other places in this file.

Reply:
should update both line 16 and 17?